.eslintrc

```diff
@@ -6,7 +6,13 @@
       "browser": true,
       "es6": true,
       "jest": true,
-      "cypress/globals": true
+      "cypress/globals": true,
+    },
+    "globals": {
+      "mount": true,
+      "render": true,
+      "toJson": true,
+      "shallow": true,
```

Comment:
Why do we have these?

Reply:
I used mount and toJson for testing the components. The other two will be needed for further testing as we introduce more codes to the repo, that is why I included them.